docs/section19/19-20.md

```diff
@@ -0,0 +1,73 @@
+**Appendix G Section:** G3.1.2.9        
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None  
+
+**Evaluation Context:** Each HeatingVentilatingAirConditioningSystem  
+
+**Applicability Checks:** 
+1. Applies to baseline HVAC system types 9 and 10.
+
+**Function Calls:**  
+1. get_baseline_system_types()    
+2. get_dict_of_zones_and_terminal_units_served_by_hvac_sys()   
+3. get_fan_object_electric_power()  
+4. baseline_system_type_compare()  
+5. get_fan_system_object_supply_return_exhaust_relief_total_kW_CFM()  
+
+## Rule Logic:   
+**Applicability Check 1**      
+- Create a list of the target system types: `target_system_types = [HVAC_SYS.SYS_9, HVAC_SYS.SYS_10]`  
+- Create dictionary of baseline system types: `baseline_system_types_dict = get_baseline_system_types(B_RMI)`  
+- Check if B-RMR is modeled with at least one air-side system that is of type 9 or 10, if yes then carry on, if no then NOT_APPLICABLE for the project: `if any(baseline_system_type_compare(system_type, target_sys_type, false) for system_type in baseline_hvac_system_dict.keys() for target_system_type in target_system_types):`  
+    - Create dictionary of hvac systems, zones, and terminal units: `dict_of_zones_and_terminal_units_served_by_hvac_sys = get_dict_of_zones_and_terminal_units_served_by_hvac_sys(B_RMI)`
+    - For each hvac system in the B_RMI: `for hvac in B_RMI...HeatingVentilatingAirConditioningSystem:` 
+        - Reset zone_hvac_in_PR_has_non_mech_cooling to False: `zone_hvac_in_PR_has_non_mech_cooling = False`  
+        - Reset zones_served_by_BL_hvac_has_non_mech_cooling boolean variable to false: `zones_served_by_BL_hvac_has_non_mech_cooling = False` 
+        - Get the baseline system type: `sys_type = list(baseline_system_types_dict.keys())[list(baseline_system_types_dict.values()).index(hvac.id)])`
+        - Check if system type is of an applicable system type:  `If (baseline_system_type_compare(sys_type, HVAC_SYS.SYS_9, false) == TRUE OR baseline_system_type_compare(sys_type, HVAC_SYS.SYS_10, false) == TRUE):`     
+            - Get the list of zones associated with the hvac system: `zone_list = dict_of_zones_and_terminal_units_served_by_hvac_sys[hvac.id]["ZONE_LIST"]`  
+            - Reset supply cfm to zero: `supply_cfm_b = 0`  
+            - Reset total_fan_power to zero: `total_fan_power = 0`   
+            - Reset fan_power_W_CFM to zero: `fan_power_W_CFM = 0`                 
+            - Get the fan system object:  `fan_sys_b = hvac.fan_system`  
+            - Get list with the hvac system's fan system's supply, return, exhaust and relief fan powers in kW: `fan_sys_kW = list(get_fan_system_object_supply_return_exhaust_relief_total_kW_CFM(B_RMI,fan_sys_b)["Fan_Power"])`  
+            - Get list with the hvac system's fan system's supply, return, exhaust and relief fan CFM: `fan_sys_CFM = list(get_fan_system_object_supply_return_exhaust_relief_total_kW_CFM(B_RMI,fan_sys_b)["Fan_Power"]["Fan_CFM"])` 
```

Comment:
I think we agreed that if there is more than one fan of any type we'd do UNDERTERMINED outcome for this system.

Reply:
OK, I am going to go with this then.

Reply:
Updated